guardian/prosemirror-elements
# feat: Add table element

## What does this change?

This PR adds the membership element, which is an element representing some HTML that contains tabular data. It's chiefly, and regularly, used by Sport to embed premier league tables, player profiles etc.

|Before|After|
|--|--|
|<img width="429" alt="Screenshot 2022-03-08 at 21 40 33" src="https://user-images.githubusercontent.com/7767575/157330437-165864bd-ed69-4bc4-9acb-11830d8ee214.png">|<img width="418" alt="Screenshot 2022-03-08 at 21 40 18" src="https://user-images.githubusercontent.com/7767575/157330458-0abe3dce-6ef3-410d-a576-d67add258098.png">|


## How to test

- We've added fixtures containing a hundred tables randomly sampled from the production database. They should pass, indicating that this change should be non-destructive for existing data.
- Add a table element. It should correctly display a table (I've included CSS in this PR to style it that's been copied from our CMS).

## Accessibility
